src/linglong/builder/source_fetcher.cpp

```diff
@@ -95,6 +95,8 @@ linglong::util::Error SourceFetcherPrivate::fetchArchiveFile()
 
     QNetworkRequest request;
     request.setUrl(url);
+    request.setAttribute(QNetworkRequest::RedirectPolicyAttribute, QNetworkRequest::NoLessSafeRedirectPolicy);
+    request.setHeader(QNetworkRequest::UserAgentHeader, "Wget/1.21.4");
```

Comment:
why？
seems unnecessary？

Reply:
![image](https://github.com/linuxdeepin/linglong/assets/29726878/6dfd5848-b142-4720-853d-1e321a2e1e00)

I don't think UA matter.

Let me have a try.

Reply:
Yep. But without set UA (use qt default UA) will download a html file, not a archive file.

Reply:
It seems that wget without UA still get archive file downloaded.

I don't have idea about what is going on here.

But set UA in qt does solve that problem. I will take that.